addons/website_event_exhibitor/views/event_exhibitor_templates_list.xml

```diff
@@ -124,7 +124,9 @@
                 <p class="m-0">We did not find any exhibitor matching your <strong t-esc="search_key"/> search.</p>
             </div>
             <div t-else="" class="alert alert-info text-center" groups="event.group_event_user">
-                <p class="m-0">Add some exhibitors to get started !</p>
+                <a target="_blank" t-att-href="'/web?#action=website_event_exhibitor.event_sponsor_action_from_event&amp;active_id=%s&amp;model=event.track&amp;view_type=kanban' % event.id">
```

Comment:
Do you really need to specify model ? Does view_type really be really supported ?

Reply:
Same, these are not needed, they are not specified now.